pysc2/agents/scripted_agent.py

```diff
@@ -49,24 +49,29 @@ def step(self, obs):
 
 class CollectMineralShards(base_agent.BaseAgent):
```

Comment:
Add this as a second agent, maybe CollectMineralShardsUnits, then add another test in tests/easy_scripted_test.py.